ui/components/ui/popover/popover.component.js

```diff
@@ -18,9 +19,47 @@ const Popover = ({
   CustomBackground,
   popoverRef,
   centerTitle,
+  headerProps = {},
+  contentProps = {},
+  footerProps = {},
 }) => {
   const t = useI18nContext();
   const showHeader = title || onBack || subtitle || onClose;
+  const Header = () => {
+    return (
+      <Box {...headerProps} className="popover-header" tag="header">
```

Comment:
I guess we should update the Box component to accept a `tag` prop as well and include the types used here as well as div?

Reply:
Removed the tag prop, that was unintentional. I say that we make a separate issue/PR for making it polymorphic.